src/lib/utils/sort.ts

```diff
@@ -26,21 +26,18 @@ const sorts: Record<
     (a: DeclarationReflection, b: DeclarationReflection) => boolean
 > = {
     "source-order"(a, b) {
-        const aSymbol = a.project.getSymbolFromReflection(a);
```

Comment:
This change is not ok. Sorting should be able to use source order even if `--disableSources` is passed.

Reply:
Thanks for flagging — oversaw this.